scylladb/config.go

```diff
@@ -0,0 +1,92 @@
+package scylladb
+
+import "github.com/gocql/gocql"
+
+type Config struct {
```

Comment:
One thing that I see missing is support for setting a TLS config.

Reply:
Added support for setting TLS config.